cmd/ingress/ingress.go

```diff
@@ -129,9 +129,12 @@ the apisix cluster and others are created`,
 
 			log.Info("version:\n", version.Long())
 
-			data, err := json.MarshalIndent(cfg, "", "\t")
+			// We should make sure that the cfg that's logged out is sanitized.
+			logCfg := cfg
```

Comment:
`cfg` here is a pointer, so we need to dereference it or it will be changed globally.

Reply:
Done